### DISCUSSION TITLE
raspapd.service 'After' field

### DISCUSSION THREAD

**Discussion by FIAV1** [RaspAP/raspap-webgui#972] [Q&A] [answered]
Hi, is there a specific reason why the `After` field in the `raspapd.service` service file is `multi-user.target` instead of `network.target`?
I tried to change it, it works perfectly and it's quicker (since hostapd finds earlier the uap0 interface in my case).

**Comment by billz** [comment, marked as answer]
@FIAV1 good question. Some users found inconsistencies with `network.target`, so `multi-user.target` was chosen to be sure that network services were up and available. This does indeed occur later in systemd's startup.

A better choice might be `After=network-online.target` ([reference](https://www.freedesktop.org/wiki/Software/systemd/NetworkTarget/))
